lib/unhangout-sockets.js

```diff
@@ -512,12 +613,25 @@ _.extend(UnhangoutSocketManager.prototype, events.EventEmitter.prototype, {
             });
         });
         this.db.events.on("change:sessionsOpen", function(event, isOpen) {
+            console.log("SESSION  : " + isOpen);
```

Comment:
Corresponding changes are indicated in this commit https://github.com/drewww/unhangout/commit/1b14335fc2d00a670645bce65506b0f6fd8b36c2